model-optimizer/mo/utils/cli_parser.py

```diff
@@ -89,12 +89,12 @@ def __call__(self, parser, namespace, values, option_string=None):
 
 class DeprecatedCanonicalizePathCheckExistenceAction(CanonicalizePathCheckExistenceAction):
     def __call__(self, parser, namespace, values, option_string=None):
-        super().__call__(parser, namespace, values, option_string)
         dep_msg = "Use of deprecated cli option {} detected. Option use in the following releases will be fatal. ".format(
             option_string)
         if 'tensorflow_use_custom_operations_config' in option_string:
             dep_msg += 'Please use --transformations_config cli option instead'
```

Comment:
I think you need to add `--mean_file` and `--mean_offsets` and recommend alternatives